src/components/RigCard.css

```diff
@@ -37,3 +37,30 @@ h3 {
   color: gray;
   margin: 0;
 }
+
+@media screen and (max-width: 73em) {
```

Comment:
Good use of the responsive layouts. They can bog down a project but it is great to see you implement so many to help the user experience.